### ISSUE TITLE
Error with seeding database - Class "DOMDocument" not found

### ISSUE THREAD

**Issue by andlearnmore** [susanBuck/e15-spring23#27]
When I run the command to refresh and seed my database (php artisan migrate:fresh --seed) I get the following error:

```
  Error 

  Class "DOMDocument" not found

  at vendor/nunomaduro/termwind/src/HtmlRenderer.php:32
     28▕      * Parses the given html.
     29▕      */
     30▕     public function parse(string $html): Components\Element
     31▕     {
  ➜  32▕         $dom = new DOMDocument();
     33▕ 
     34▕         if (strip_tags($html) === $html) {
     35▕             return Termwind::span($html);
     36▕         }

      +23 vendor frames 
  24  artisan:35
      Illuminate\Foundation\Console\Kernel::handle()
```

I am not even sure where to begin with this and would appreciate help! My database is empty, which is the only clue I have.
Thanks!
-Anne

**Comment by andlearnmore** [comment]
I think I got it working again following this: https://stackoverflow.com/questions/14395239/class-domdocument-not-found#:~:text=If%20you%20cannot%20see%20DOM,from%20source%20to%20remedy%20that.&text=i%20found%20%2D%2Ddisable%2Ddom,what%20should%20i%20do%20next%3F&text=That%20depends%20on%20your%20system.

**Comment by susanBuck** [comment]
Hi @andlearnmore - Glad you got this figured out. Out of curiosity, has your migrations/seeds been running fine previously? 

Back when we installed Composer, we ran some commands ([notes here](https://hesweb.dev/e15/notes/php/composer#composer-pre-install-extensions)) that should have included the DOM extension in question. 

Given that, it's a little odd you'd just start encountering this issue now, so I'm curious to try and understand why/how.

**Comment by andlearnmore** [comment]
Hi @susanBuck - yes, the migrations and seeds had been running fine until this point. I wonder if something happened when I installed the phpMyAdmin because it was after that that I started having the trouble. In that process, I did get an extra screen from the ones in the notes/video (that I unfortunately failed to screen-capture). It said that some updates were needed--and I just pressed ok without selecting any of the 3 options it gave me of things to update.
